shannon/testnet/genesis.json

```diff
@@ -0,0 +1,742 @@
+{
+  "app_name": "poktrolld",
+  "app_version": "0.0.11-rc",
+  "genesis_time": "2024-11-19T22:52:46.166451Z",
+  "chain_id": "poktroll",
+  "initial_height": 1,
+  "app_hash": null,
+  "app_state": {
+    "application": {
+      "applicationList": [],
+      "application_list": [],
+      "params": {
+        "max_delegated_gateways": "7",
+        "min_stake": {
+          "amount": "100000000",
+          "denom": "upokt"
+        }
+      }
+    },
+    "auth": {
+      "params": {
+        "max_memo_characters": "256",
+        "tx_sig_limit": "7",
+        "tx_size_cost_per_byte": "10",
+        "sig_verify_cost_ed25519": "590",
+        "sig_verify_cost_secp256k1": "1000"
+      },
+      "accounts": [
+        {
+          "@type": "/cosmos.auth.v1beta1.BaseAccount",
+          "address": "pokt1f0c9y7mahf2ya8tymy8g4rr75ezh3pkklu4c3e",
```

Comment:
What are all these accounts?

Reply:
@Olshansk the `config.yml` I used to generate them is attached to the first PR message. I can also share the zip with all secrets, including the mnemonics for all accounts.
Basically, we have the following accounts:
1 PNF, 1 Faucet, 5 validators.

Reply:
Sgtm, just have a tiny README with the details